manifests/repo/apt/cloudsmith.pp

```diff
@@ -0,0 +1,30 @@
+# erlang cloudsmith apt repo
+class erlang::repo::apt::cloudsmith (
+  String $ensure      = $erlang::repo::apt::ensure,
+  String $location    = 'https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-erlang/deb/debian',
+  # trusty, xenial, bionic, etc
+  String $release     = downcase($facts['os']['distro']['codename']),
+  String $repos       = 'main',
+  String $key         = 'A16A42516F6A691BC1FF5621E495BB49CC4BBE5B',
+  String $key_source  = 'https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-erlang/gpg.E495BB49CC4BBE5B.key',
+  Optional[Variant[Numeric, String]] $pin = $erlang::package_apt_pin,
+) inherits erlang {
```

Comment:
This inheritance is at best dubious, but it seems all classes from this module have this strange construct and I am not a user of this module so maybe it serve some purpose?

I feel like removing the inherit and all parameters can to the same thing by just `include erlang` and then using `$erlang::ensure`, `$erlang::location` and so on…

Reply:
Changed it for cloudsmith. For remaining parts it most likly should be a separate PR to not mix context :)